test/unit/event/owner/hcs-did-create-did-owner-event.test.ts

```diff
@@ -0,0 +1,144 @@
+import { PrivateKey } from "@hashgraph/sdk";
+import { DidError, Hashing, HcsDidCreateDidOwnerEvent, HcsDidEventTargetName } from "../../../../dist";
+
+describe("HcsDidCreateDidOwnerEvent", () => {
+    const privateKey = PrivateKey.fromString(
+        "302e020100300506032b6570042204209044d8f201e4b0aa7ba8ed577b0334b8cb6e38aad6c596171b5b1246737f5079"
```

Comment:
Can this private key be generated at run time ?

Reply:
Want a known input value, so that unit test can inspect the output to check for a known output.

The key is not associated with any account.